sklearn/svm/_libsvm_sparse.pyx

```diff
@@ -1,8 +1,7 @@
 import  numpy as np
-cimport numpy as cnp
 from scipy import sparse
```

Comment:
Generally should we move python imports bellow cython imports?

Reply:
While I grouped it better in a few places that I touched anyways, I'd like to keep this PR at the current size.